runtime/bcutil/jsrinliner.c

```diff
@@ -1159,11 +1159,11 @@ evaluateCodeBlock (I_32 hasRET, J9JSRICodeBlock ** blockParentPointer, U_32 star
 					stackError |= popStack (jsrData);
 					if (args[step] == '[') {
 						while (args[++step] == '[');
-						if (args[step] != 'L') {
+						if (J9_IS_NOT_OBJECT_AND_NOT_VALUETYPE(args[step])) {
```

Comment:
This change is unnecessary as Q types are at least a JDK > 12 feature (not clear when they will ship) and JSRs are only allowed in classfiles that are older than JDK 7 (v.51) as they aren't compatible with StackMapTables.

All that to say, we shouldn't update the JSRInliner for Q types.